components/02-molecules/social-links/_social-links.scss

```diff
@@ -0,0 +1,15 @@
+@use '../../01-atoms/atoms';
+@use '../../00-tokens/tokens';
+
+.social-links__list {
+  @include atoms.list-reset;
+
+  display: flex;
+  gap: var(--size-spacing-6);
```

Comment:
The gap size looks different for mobile and desktop, let's use a media query to support that.

Reply:
The gap size is fixed.